src/web/components/elements/YouTubeComponent.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { Caption } from '@root/src/web/components/Caption';
+import { YouTubeEmbed } from '@root/src/web/components/YouTubeEmbed';
+
+type Props = {
+    children: JSX.Element | JSX.Element[];
+};
+
+const Margins = ({ children }: Props) => (
+    <div
+        className={css`
+            margin-top: 16px;
+            margin-bottom: 12px;
```

Comment:
I'm not sure why we would need a margin bottom, I'd hope all elements handle their top margins only to keep consistency?

Reply:
The thing here is that video element has extra padding vs. normal images.

### Main media
![2020-01-13 10 07 58](https://user-images.githubusercontent.com/1336821/72247546-feb43b80-35ec-11ea-9d4e-c4d3876648de.gif)

### In body
![2020-01-13 10 10 05](https://user-images.githubusercontent.com/1336821/72247547-feb43b80-35ec-11ea-9eae-2f8dcff7eee9.gif)

So to replicate this these margins work better local to the video (rather than setting margins on paragraphs dynamically based on the previous element, yikes)

Annoyingly, the _in body_ video element only has extra spacing at the bottom so for parity we'd need to break these margins out into two margin wrappers, one for main media with the current margins and another for in body videos with different margins.

Reply:
I would remove the extra margins I think! I imagine that's a mistake and it should be the same as images.

Reply:
Yep: always assume mistake, when something looks off/inconsistent!

Reply:
🎉 Less code!